index.html

```diff
@@ -2537,6 +2537,16 @@ <h2>Production and Consumption</h2>
         </li>
       </ol>
 
+      <figure id="production-consumption">
+        <img style="margin: auto; display: block;"
+          src="diagrams/diagram-production-consumption.png" alt="
+Diagram illustrating how representations of the data model are produced
+and consumed, including in JSON, JSON-LD, and CBOR.">
```

Comment:
We'll need to fix this description. Imagine you can't see, and you read this... have you learned the same thing that someone with sight has? If not, the description needs improvement. :)